birdhouse/components/stac/default.env

```diff
@@ -0,0 +1,4 @@
+export STAC_POSTGRES_USER=${POSTGRES_PAVICS_USERNAME}
+export STAC_POSTGRES_PASSWORD=${POSTGRES_PAVICS_PASSWORD}
+export STAC_PGUSER=${POSTGRES_PAVICS_USERNAME}
+export STAC_PGPASSWORD=${POSTGRES_PAVICS_PASSWORD}
```

Comment:
Missing variable registration for expansion and the dependant components (only `twitcher`, `magpie`, `proxy` in this case). 
See other comment for details.